doc/design/README.md

```diff
@@ -0,0 +1,43 @@
+# Operator design
+
+# Controllers and their managed resources
+
+
+- **lvmcluster-controller:** Running in the operator deployment, it will create all resources that are don't require information from the node. When applicable, the health of the underlying resource is updated in the LVMCluster status and errors are also exposed as events. Overall success also passed on as an event.:
```

Comment:
This will be a separate pod, not part of the operator deployment.

Reply:
This is the operator deployment. This is not referring to the CSI driver.